modules/ROOT/pages/oauth-policy-implementation-concept.adoc

```diff
@@ -12,55 +12,85 @@ image::oauth-implementation-federated.png[]
 . The federated policy validates the token against the OAuth provider. To improve performance, Mule caches the call to the authorization server, which is only performed once during the lifetime of the token.
 +
 When OpenId, PingFederate, or OAuth 2.0 Access Token Enforcement Using External Provider policies are in effect and your API is deployed under Mule 4, the client ID returned by the OAuth provider is validated by the Client Id Enforcement policy.
-. All fulfilled requests in the flow are enriched with the information provided by the OAuth Provider when running under Mule 4, but information is not returned to the user. In releases earlier than Mule 4, this information was returned to the user.
-
-Possible responses are:
-
-*Mule 4.0.0 or later*
-
-* 200
-+
-The token was successfully validated.
+. As opposed to Mule 3, in Mule 4, all fulfilled requests in the flow are enriched with the information provided by the OAuth Provider, but this information is not returned to the user unless explicitly configured in the policy by enabling the *Expose Headers* checkbox:
 +
-* 400 Bad Request
+image::oauth-policy-implementation-concept-7ad82.png[]
 +
-This Bad Request error can be caused by: no token sent, the token was sent both as query param and header, or the token was sent as header but does not have prefix "Bearer".
+This flag blacks out `x-rate-limit` headers from the response. +
```

Comment:
What does "blacks out" mean in this context? the headers are "removed" or "excluded" from the response?
Is it possible to use a different word that clarifies what happens with the response headers?